### PR TITLE
Infections

### PR DESCRIPTION
**Pull request by rw251**
- Took the Bristol infection code set and split it into its component parts
- Supplemented the Read v2 codes with SNOMED from a hierarchy search of the term browser
- Found missing codes and recorded prevalence
- Used the template logic to create an infections code set (v2) which is the union of the individual code sets
- Also added logic so that the AUTOGERATED FILE warning starts with a '#' and then treat any rows that start with a '#' as a comment and so ignore them from the sql creation

### CODE REVIEW

File: shared/clinical-code-sets/conditions/genital-tract-infections/1/README.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+| Date       | Practice system | Population | Patients from ID | Patient from code |
+| ---------- | --------------- | ---------- | ---------------: | ----------------: |
+| 2024-07-09 | EMIS            | 2533608    |   215633 (8.51%) |    215616 (8.51%) |
+| 2024-07-09 | TPP             | 201974     |    14831 (7.34%) |     14831 (7.34%) |
+| 2024-07-09 | Vision          | 335755     |    26957 (8.03%) |     26902 (8.01%) |
+
+## Audit log
+
+- Find_missing_codes last run 2024-07-05
+
```
Comment (Georgetilston) [inline shared/clinical-code-sets/conditions/genital-tract-infections/1/README.md:24]: is the text below meant to be in this README file?

File: shared/clinical-code-sets/conditions/uti/2/uti.ctv3.txt [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+K01x2	Nephrotic syndrome in malaria
+K0y0.	Late syphilis of kidney
+K10..	Infection of kidney
+K100.	Chronic pyelonephritis
+K1001	Chronic pyelonephritis with medullary necrosis
+K1003	Chronic pyonephrosis
+K1004	Non-obstructive reflux-associated chronic pyelonephritis
+K1006	Calculous pyelonephritis
+K100z	Chronic pyelonephritis NOS
+K101.	Acute pyelonephritis
```
Comment (Georgetilston) [inline shared/clinical-code-sets/conditions/uti/2/uti.ctv3.txt:25]: the README says this code set excludes pyelonephritis but there are a few codes here

File: shared/clinical-code-sets/conditions/genital-tract-infections/1/README.md (hunk 1 above)
Comment (rw251) [reply shared/clinical-code-sets/conditions/genital-tract-infections/1/README.md]: Good spot. Now removed.

File: shared/clinical-code-sets/conditions/uti/2/uti.ctv3.txt (hunk 2 above)
Comment (rw251) [reply shared/clinical-code-sets/conditions/uti/2/uti.ctv3.txt]: Code removed and prevalence updated.

**Review by Georgetilston — APPROVED** [review]

**Pull request merged**